packages/api/db/migrations/20200707125043_create_texts_table.ts

```diff
@@ -0,0 +1,16 @@
+import Knex from "knex";
+
+export const up = async (knex: Knex) => {
+    return knex.schema.createTable("texts", table => {
+        table.increments("id");
+        table.string("text").notNullable();
+        table.boolean("ordered");
+        table.boolean("tutorial");
+        table.integer("difficulty").notNullable();
+        table.integer("author").notNullable();
```

Comment:
Btw, I think I forgot but it would also be cool if we can have a title for texts as a string. For example when I want to display the tutorials on the tutorials page I would like to have a Title for each so people know what it is about also for non easy tests we could have quotes from books and having a title would be nice.